_posts/2018-06-14-fio-tutorial.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Running FIO with pmem engines
+author: lukaszstolarczuk
+layout: post
+identifier: fio_tutorial
+---
+
+When we want to check performance of our library, either see if there was any regression or if our tweaks did the good job, we execute benchmarks. One of them is FIO. It helps us simulate synthethic traffic of reads and writes to pmem device. In this blog post we'll try to introduce this tool and explain how we use it.
+
+### Preparing environment
+For starters, links to needed software:
+* [FIO][1fioLink]
+* [NDCTL][ndctlLnk]
+And of course, to execute any FIO tests using pmem engines you need PMDK installed in your system.
+
+Tests can be run on [emulated pmem][13rde117] or real hardware (devices introduced as NVDIMM Firmware Interface Table (NFIT), according to ACPI Specification v6.0+). Either way you have to prepare device(s) before test execution.
```

Comment:
I'm not sure what you mean.
devices introduced *by* NFIT?

maybe: "devices presented by NFIT", "displayed", or something? not sure what's the correct word here.